sunbird-insurance-default.properties

```diff
@@ -0,0 +1,8 @@
+##---------------------------------Sunbird-RC Plugin Insurance Credential Configurations------------------------------------------------------
```

Comment:
I thought about this segregation, now I think it will be better to maintain only 2 property files.
1. certify-default.properties
2. certify-plugin-default.properties

Anyways for each usecase, we might create a namespace and the profile part which is default will be changed to insurance etc..

But lets atleast segregate the certify properties and the properties used by plugins into 2 different files, so when the certify is upgraded and plugin is same, it is easier to upgrade the properties.

Inside certify-plugin-default.properties we can have one section of each plugin and usecase.

Reply:
moved them to certify-plugin-default.properties file